plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/wst/server/ui/cnf/ServerModule.java

```diff
@@ -185,5 +191,45 @@ public void restart(){
 		Class<?>[] classes = {TreeItem.class, ServersView2.class};
 		return classes;
 	}
+	
+	/**
+	 * Get submodules
+	 * @return List of submodules
+	 */
+	public List<ServerModule> getSubmodules() {
```

Comment:
Please compare naming with eclipse or jbosstools implementation and update it accordingly.

Reply:
Renamed according to Eclipse.